Add debugger troubleshooting guide (#1552)
ballerina-platform/ballerina-www: website/two-column-pages/docs/learn/tools-ides/intellij-plugin/using-intellij-plugin-features.md

![Debug Ballerina programs](../images/debug-ballerina-intellij.gif)

> ## Troubleshooting

Reviewer: ```suggestion
### Troubleshooting
```

> - Stepping over code lines in non-blocking paths (eg: action invocations) will not pause VM on next line

Reviewer 2: ```suggestion
- You need to manually add a breakpoint to the next line when stepping over code lines in non-blocking paths (e.g., action invocations). Otherwise, it will not pause the VM on the next line by default.
```

    - workaround: manually put a breakpoint to next line

> - There are some cases where stepping over gives unexpected behavior

Reviewer 2: ```suggestion
- However, this will not happen in certain instances. For example, when there are multiple workers and a `wait` expression waiting for them, even though stepping over hits and passes the waiting line in the source, workers might not yet be executed.
```
